### DISCUSSION TITLE
Cruise-control with operator

### DISCUSSION THREAD

**Discussion by therus000** [strimzi/strimzi-kafka-operator#10559] [Q&A]
Good day!
i deploy cruise-control with operator like this 
'''
  cruiseControl: 
      config:
      # Note that `goals` must be a superset of `default.goals` and `hard.goals`
      goals: >
        com.linkedin.kafka.cruisecontrol.analyzer.goals.CpuCapacityGoal,      
        com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal,
        com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundCapacityGoal,
        com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundCapacityGoal,
        com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,
        com.linkedin.kafka.cruisecontrol.analyzer.goals.MinTopicLeadersPerBrokerGoal,
        com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal, 
'''
I got a question. Is there a way to run this once a week only? Maybe some config to do that once a week?

**Comment by scholzj** [comment]
If you wanna run a rebalance periodically, you can for example create some CronJob to create the KafkaRebalance resource to do the rebalance for you.

**Reply by therus000** [reply]
is there an example how to do that?

**Reply by scholzj** [reply]
I don't have any.
